pkg/reconciler/instances/istio/actions/performer.go

```diff
@@ -363,6 +377,41 @@ func getTargetVersionFromIstioChart(workspace chart.Factory, branch string, isti
 	return "", errors.New("Target Istio version could not be found neither in Chart.yaml nor in helm values")
 }
 
+func getTargetProxyV2PrefixFromIstioChart(workspace chart.Factory, branch string, istioChart string, logger *zap.SugaredLogger) (string, error) {
+	ws, err := workspace.Get(branch)
+	if err != nil {
+		return "", err
+	}
+
+	istioHelmChart, err := loader.Load(filepath.Join(ws.ResourceDir, istioChart))
+	if err != nil {
+		return "", err
+	}
+
+	mapAsJSON, err := json.Marshal(istioHelmChart)
+	if err != nil {
+		return "", err
+	}
+
+	var chartValues chartValues
+	err = json.Unmarshal(mapAsJSON, &chartValues)
+	if err != nil {
+		return "", err
+	}
+
+	if err!=nil{
+		return "", err
+	}
+
+	prefix := chartValues.Global.Images.IstioProxyV2.ContainerRegistryPath + "/" + chartValues.Global.Images.IstioProxyV2.Directory
```

Comment:
Could you use `fmt.Sprintf()` for string formatting instead of raw concatenation?